api/src/main/resources/messages.properties

```diff
@@ -89,5 +89,5 @@ ${project.parent.artifactId}.report.visits.category7.label=Pregnant women
 ${project.parent.artifactId}.report.visits.category8.label=Family planing clients
 ${project.parent.artifactId}.report.visits.category9.label=Other adults
 ${project.parent.artifactId}.report.visits.total.label=Total
-${project.parent.artifactId}.report.familyPlanning.reportName=Family Planning
-${project.parent.artifactId}.report.familyPlanning.reportDescription=MSPP report required by Haitian MoH to any clinic in Haiti
+${project.parent.artifactId}.report.familyPlanning.reportName=MSPP: Family Planning
```

Comment:
Unfortunately the full-colon causes inconsistencies at runtime when the view the report via the web interface. So let's remove it.

Reply:
Done